### DISCUSSION TITLE
QMUIKit能否扩展支持ViewController的强制旋转？

### DISCUSSION THREAD

**Discussion by KatyChenLu** [Tencent/QMUI_iOS#1571] [Q&A]
假设工程仅支持横屏，但某些ViewController必须使用竖屏，此时通过QMUI配置旋转方向是不生效的，能否支持一下呢？

**Comment by MoLice** [comment]
Demo 里应该已经有示例了？`qmui_setNeedsUpdateOfSupportedInterfaceOrientations`
<img width="447" alt="image" src="https://github.com/user-attachments/assets/84f3b886-2053-45ab-b34f-eb236fb6906f">
